modules/vcf2maf/1.0/vcf2maf.smk

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Bruno Grande
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CONFIG`
+CONFIG = config["lcr-modules"]["vcf2maf"]
+LOG = "/logs/" + op._session.launched_fmt
```

Comment:
This is different than I'm used to seeing. Can you document what's happening in a comment above this line?

Reply:
okay!